assets/js/theme/common/product-page-details.js

```diff
@@ -1,51 +1,27 @@
 import utils from '@bigcommerce/stencil-utils';
+import ProductDetailsBase, { optionChangeDecorator } from './product-details-base';
 import 'foundation-sites/js/foundation/foundation';
 import 'foundation-sites/js/foundation/foundation.reveal';
 import ImageGallery from '../product/image-gallery';
 import modalFactory, { showAlertModal, modalTypes } from '../global/modal';
-import _ from 'lodash';
-import Wishlist from '../wishlist';
+import { isEmpty, isPlainObject } from 'lodash';
 import { normalizeFormData } from './utils/api';
-import { initRadioOptions } from './aria';
 import { isBrowserIE, convertIntoArray } from './utils/ie-helpers';
 
-const optionsTypesMap = {
-    INPUT_FILE: 'input-file',
-    INPUT_TEXT: 'input-text',
-    INPUT_NUMBER: 'input-number',
-    INPUT_CHECKBOX: 'input-checkbox',
-    TEXTAREA: 'textarea',
-    DATE: 'date',
-    SET_SELECT: 'set-select',
-    SET_RECTANGLE: 'set-rectangle',
-    SET_RADIO: 'set-radio',
-    SWATCH: 'swatch',
-    PRODUCT_LIST: 'product-list',
-};
-
-export default class ProductDetails {
+export default class ProductPageDetails extends ProductDetailsBase {
```

Comment:
I think we should not attach class name to the Page, because generally product details can appear on any page

Reply:
agree